services.json.tmpl

```diff
@@ -0,0 +1,9 @@
+[
+    {"name": "Mesos", "id": "mesos", "check": "mesos", "path": "/mesos"},
+    {"name": "Marathon", "id": "marathon", "check": "marathon", "path": "/marathon"},
+    {"name": "Consul", "id": "consul", "check": "consul", "path": "/consul/"}{{with service "traefik" "any"}},
+    {"name": "Traefik", "id": "traefik", "check": "traefik-admin", "path": "/traefik"}{{end}}{{with service "chronos" "any"}},
+    {"name": "Chronos", "id": "chronos", "check": "chronos", "path": "/chronos"}{{end}}{{with service "kibana" "any"}},
+    {"name": "Kibana", "id": "kibana", "check": "kibana", "path": "/kibana"}{{end}}{{with service "elasticsearch" "any"}},
+    {"name": "Elasticsearch", "id": "elasticsearch", "check": "elasticsearch", "path": "/elasticsearch"}{{end}}
```

Comment:
I think I see what's going on here, but just to confirm: it looks like this is adding commas iff a service is present, yes? That means we won't get a trailing comma and invalid JSON?